files/scripts/update-cluster.sh

```diff
@@ -3,9 +3,9 @@ set -eu
 
 function patchCluster {
   local configLocation=$1
-  local authToken=$2
+  local authToken=$PROVISION_USER_AUTH_TOKEN
   echo "Patching cluster"
-  waitService "http://graphdb-cluster-proxy:7200/proxy/ready" "$authToken"
+  waitService "http://graphdb-cluster-proxy:7200/proxy/ready"
```

Comment:
hm, what if I don't have an external proxy deployed? the same applies to removeNodes and addNodes?

Reply:
I'm looking into it

Reply:
So the reason it's using the external proxy is because the removing of nodes and patching the cluster both return "Followers cannot directly process transactions"  I am currently making a script that finds the leader on it's own and sends the request to it.